mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -34,36 +36,58 @@ CLASS avx net/minecraft/entity/player/PlayerEntity
 	FIELD e sleepTimer I
 	FIELD f spawnPosition Lev;
 	FIELD g spawnForced Z
+	METHOD <init> (Lbhh;Lcom/mojang/authlib/GameProfile;)V
+		ARG 1 world
+		ARG 2 profile
 	METHOD a sendTradeOffers (ILbgl;IIZ)V
 		ARG 1 syncId
+		ARG 2 offers
+		ARG 3 levelProgress
+		ARG 4 experience
+		ARG 5 leveled
 	METHOD a openContainer (Lahd;)Ljava/util/OptionalInt;
 	METHOD a addCritParticles (Laif;)V
+		ARG 1 target
 	METHOD a interact (Laif;Lagz;)Laha;
 		ARG 1 entity
+		ARG 2 hand
 	METHOD a setMainHand (Lain;)V
+		ARG 1 absoluteHand
```

Comment:
Is this actually `HandSide` or what

Reply:
It should be `hand`, even if it's an `AbsoluteHand`.

Reply:
Hmm, no. `absoluteHand` is left or right. This changes the `mainHand` to either the left or right hand, so `MainHand` has nothing to do with `AbsoluteHand`.